api/end_device.proto

```diff
@@ -56,13 +56,10 @@ message Session {
 // This is used internally by the Network Server and is read only.
 message MACParameters {
   option (gogoproto.populate) = false;
+  reserved 2, 3;
 
   // Maximum EIRP (dBm).
   float max_eirp = 1 [(gogoproto.customname) = "MaxEIRP"];
-  // Uplink dwell time is set (400ms).
-  bool uplink_dwell_time = 2;
```

Comment:
Is it OK to deprecate these like this or should I add logic, which would handle the migration to the new format?